src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -216,6 +217,48 @@ private void OpenResultsEvent(object index, bool isMouseClick)
                             Hide();
                         }
 
+                        // Check information file for version mismatch
+                        try
+                        {
+                            // UserSelectedRecord
+                            var userSelectedRecordItemData = new UserSelectedRecord.UserSelectedRecordItem();
+                            var userSelectedRecordItemStorage = new WoxJsonStorage<UserSelectedRecord.UserSelectedRecordItem>(_userSelectedRecordStorage.GetFileName());
+                            userSelectedRecordItemStorage.Load();
+
+                            if (userSelectedRecordItemData != null)
+                            {
+                                if (userSelectedRecordItemStorage.CheckVersionMismatch(userSelectedRecordItemData, _userSelectedRecordStorage.GetVersionMismatch()))
```

Comment:
Can there be a situation where userSelectedRecordItemStorage._storageHelper.ClearCache has different value than _userSelectedRecordStorage._storageHelper.ClearCache. I believe not? In that case why creating a copy of the storage, can we just use the field member storage?

Reply:
Done.